### PR TITLE
Added beginning of water dream

### PR DESCRIPTION
**Pull request by Trafire**
Please review this, @casey-collab 

Thanks

### CODE REVIEW

File: english/virtualreality/voice/strangeVoice.md [hunk 1]
```diff
@@ -13,4 +13,19 @@ and your right hand looks like a hotdog.
...
+
+The other droplets around you seem content to fall as preordained by physical forces of the world. 
+Perhaps they once fought to control their destiny as well, but today they were firmly entrenched in the water cycle.
+ Falling from tens of thousands of feet to the ground was an ordinary Saturday Morning. 
+
+But this was different for you, a first rain, you only have one of those. 
+You peer down below you and see you have two choices, left or right. The Fish Sanctuary or The pig farm?
+
+[The Pig Farm](../../water-dreams/pig-farm.md)
+[The Fish Sanctuary](../../water-dreams/fish-sanctuary.md)
```
Comment (casey-collab) [inline english/virtualreality/voice/strangeVoice.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/water-dreams/fish-sanctuary.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Ploop! What a rush! hey guys! you scream out too all the other waterdrops in the sacturary,
```
Comment (casey-collab) [inline english/water-dreams/fish-sanctuary.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/water-dreams/pig-farm.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+You hit with a splat, your entire body folds in on itself and mixes with the pigs dinner,
```
Comment (casey-collab) [inline english/water-dreams/pig-farm.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/virtualreality/voice/strangeVoice.md (hunk 1 above)
Comment (Trafire) [inline english/virtualreality/voice/strangeVoice.md]: Fixed, thanks!

File: english/water-dreams/fish-sanctuary.md (hunk 2 above)
Comment (Trafire) [inline english/water-dreams/fish-sanctuary.md]: Fixed, thanks

File: english/water-dreams/pig-farm.md (hunk 3 above)
Comment (Trafire) [inline english/water-dreams/pig-farm.md]: Fixed, thanks

**Comment by Trafire** [comment]
Please review this, @casey-collab

Thanks

File: english/virtualreality/voice/strangeVoice.md [hunk 4]
```diff
@@ -13,4 +13,20 @@ and your right hand looks like a hotdog.
...
+warping and shedding small pieces of yourself as you are buffeted by the air currents.
+
+The other droplets around you seem content to fall as preordained by physical forces of the world. 
+Perhaps they once fought to control their destiny as well, but today they were firmly entrenched in the water cycle.
+ Falling from tens of thousands of feet to the ground was an ordinary Saturday Morning. 
+
+But this was different for you, a first rain, you only have one of those. 
+You peer down below you and see you have two choices, left or right. The Fish Sanctuary or The pig farm?
+
+[The Pig Farm](../../water-dreams/pig-farm.md)
```
Comment (casey-collab) [inline english/virtualreality/voice/strangeVoice.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/virtualreality/voice/strangeVoice.md [hunk 5]
```diff
@@ -13,4 +13,20 @@ and your right hand looks like a hotdog.
...
+
+[The Fish Sanctuary](../../water-dreams/fish-sanctuary.md)
```
Comment (casey-collab) [inline english/virtualreality/voice/strangeVoice.md]: This link is also invalid.

File: english/water-dreams/fish-santuary/fish-santuary.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+Ploop! What a rush! hey guys! you scream out too all the other waterdrops in the sacturary,
```
Comment (casey-collab) [inline english/water-dreams/fish-santuary/fish-santuary.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

File: english/water-dreams/pig-farm/pig-farm.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+You hit with a splat, your entire body folds in on itself and mixes with the pigs dinner,
```
Comment (casey-collab) [inline english/water-dreams/pig-farm/pig-farm.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Trafire** [comment]
Please review this, @casey-collab

Thanks

File: english/virtualreality/voice/strangeVoice.md [hunk 8]
```diff
@@ -13,4 +13,19 @@ and your right hand looks like a hotdog.
...
+The other droplets around you seem content to fall as preordained by physical forces of the world. 
+Perhaps they once fought to control their destiny as well, but today they were firmly entrenched in the water cycle.
+ Falling from tens of thousands of feet to the ground was an ordinary Saturday Morning. 
+
+But this was different for you, a first rain, you only have one of those. 
+You peer down below you and see you have two choices, left or right. The Fish Sanctuary or The pig farm?
+
+[The Pig Farm](../../water-dreams/pig-farm/pig-farm.md)
+
+[The Fish Sanctuary](../../water-dreams/fish-sanctuary/fish-sanctuary.md)
```
Comment (casey-collab) [inline english/virtualreality/voice/strangeVoice.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/water-dreams/fish-santuary/fish-santuary.md (hunk 6 above)
Comment (casey-collab) [inline english/water-dreams/fish-santuary/fish-santuary.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Trafire** [comment]
Please review this, @casey-collab

Thanks

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**